src/parsers/perf/perfparser.cpp

```diff
@@ -503,6 +503,33 @@ struct SymbolData
     }
 };
 
+struct CallerCalleeLocation
+{
+    QString symbol;
+    QString binary;
+
+    bool operator<(const CallerCalleeLocation& location) const
```

Comment:
Qt coding style has the & next to the variable name: https://wiki.qt.io/Qt_Coding_Style